src/oaklib/datamodels/fhir.yaml

```diff
@@ -128,10 +128,18 @@ classes:
   CodeSystemProperty:
```

Comment:
@cmungall In the future if I need to edit the LinkML yaml, do I need to manually re convert the dataclass? I think I can find how to do that in the LinkML docs, but just want to check and also see if there's any special considerations  / steps I should take when doing this in OAK.

Reply:
Yes. This is easy. Like most of our repos, everything is coordinated by the Makefile. If you edit the yaml, just make the python target